### PR TITLE
SAK-46944 SIGNUP Provide a counter in the forms title

### PR DESCRIPTION
**Pull request by CesarMarquina**
https://sakaiproject.atlassian.net/browse/SAK-46944

### CODE REVIEW

File: signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java [hunk 1]
```diff
@@ -287,6 +287,24 @@ public SignupTimeslot getTimeslot(Long timeslotId) {
 		return null;
 	}
 
+	/**
+	 * This method will obtain the the number of participants signed
+	 * 
+	 * @return a int
+	 */
+	public int getParticipantsNum() {
+		int retorno = 0;
```
Comment (feralvarez95) [inline signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java:296]: change by something like numParticipants

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java [hunk 2]
```diff
@@ -1458,4 +1459,41 @@ public String getCurrentUserEmailAddress() {
 	public String getDisplayTimeFromInstant(Instant instant) {
 		return signupMeetingService.getUsersLocalDateTimeString(instant);
 	}
+	
+	public int getNumSigned(List<TimeslotWrapper> lista, int partOrWait) {
+		int retorno = 0;
```
Comment (feralvarez95) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java:1464]: Same as before, change by numSigned or something like that

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java [hunk 3]
```diff
@@ -1458,4 +1459,41 @@ public String getCurrentUserEmailAddress() {
 	public String getDisplayTimeFromInstant(Instant instant) {
 		return signupMeetingService.getUsersLocalDateTimeString(instant);
 	}
+	
+	public int getNumSigned(List<TimeslotWrapper> lista, int partOrWait) {
```
Comment (feralvarez95) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java:1463]: change lista by timeslotList

File: signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java [hunk 4]
```diff
@@ -287,6 +287,24 @@ public SignupTimeslot getTimeslot(Long timeslotId) {
...
+	 * 
+	 * @return a int
+	 */
+	public int getParticipantsNum() {
+		int retorno = 0;
+		if (signupTimeSlots == null) {
+			return 0;
+		}
+
+		for (SignupTimeslot timeslot : signupTimeSlots) {
```
Comment (feralvarez95) [inline signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java:301]: You can use here Java lambda

**Review by feralvarez95 — CHANGES_REQUESTED** [review]

File: signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java (hunk 1 above)
Comment (CesarMarquina) [reply signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java]: Done

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java (hunk 2 above)
Comment (CesarMarquina) [reply signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java]: Done

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java (hunk 3 above)
Comment (CesarMarquina) [reply signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java]: Done

File: signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java (hunk 4 above)
Comment (CesarMarquina) [reply signup/api/src/java/org/sakaiproject/signup/model/SignupMeeting.java]: Done, thanks.

**Review by feralvarez95 — APPROVED** [review]

File: signup/tool/src/bundle/messages.properties [hunk 5]
```diff
@@ -828,6 +830,16 @@ tab_event_action = Action
 
 tab_event_availability = Status
 
+tab_event_signed=Signed
+
+tab_event_total_attendees=Participants (Total\: {0})
+
+tab_event_total_waiting_list=Wait List (Total\: {0})
```
Comment (ern) [inline signup/tool/src/bundle/messages.properties:837]:
```suggestion
tab_event_total_waiting_list=Waiting (Total\: {0})
```

**Review by ern — DISMISSED** [review]
Reviewed on Core call that the "lock" label should be refactored

**Review by juanjmerono — APPROVED** [review]

**Pull request merged**